Makefile

```diff
@@ -1,7 +1,7 @@
 TOOLS ?= autonomy/tools:b4e3778
 
 # TODO(andrewrynhard): Move this logic to a shell script.
-BUILDKIT_VERSION ?= v0.5.0
+BUILDKIT_VERSION ?= master@sha256:455f06ede03149051ce2734d9639c28aed1b6e8b8a0c607cb813e29b469a07d6
```

Comment:
I can update our current deployment to match this.

Reply:
Updated and it looks like this works!

Reply:
oh, that's awesome!

let me rebase and squash it. still waiting on dockerfile PR, and we can update `Dockerfile` slug back to upstream